src/components/MovieCard.js

```diff
@@ -0,0 +1,17 @@
+import React from "react"
```

Comment:
./src/components/MovieCard.js
  Line 3:22:  'Router' is defined but never used  no-unused-vars
  Line 4:5:   'Route' is defined but never used   no-unused-vars

Reply:
ok, rozumiem;)